scalatest/src/test/scala/user/org/mockito/PrefixExpectationsTest.scala

```diff
@@ -69,7 +56,7 @@ class PrefixExpectationsTest extends AnyWordSpec with Matchers with ArgumentMatc
       }
 
       "prevent misuse of 'expect no calls _to_' instead of 'on' when mock object access looks like a method call" in {
-        val exception = intercept[MissingMethodInvocationException] {
+        val exception = intercept[NotAMockException] {
```

Comment:
Ah I didn't know `the[E] thrownBy` can expose the exception - wanna use it here instead of `intercept`?

```suggestion
        val exception = the[NotAMockException] thrownBy {
```

Reply:
Sure